data/json/effects_on_condition/misc_effect_on_condition.json

```
@@ -448,5 +448,13 @@
     "id": "EOC_CONDITION_XE_SUNSET_MORALE_OVERRIDES",
     "condition": { "math": [ "0 == 0" ] },
     "effect": [  ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_REVEAL_HELI",
+    "effect": [
+      { "u_location_variable": { "context_val": "sunken_heli" }, "target_params": { "om_terrain": "sunken_heli", "z": 0 } },
```

```suggestion
      { "u_location_variable": { "context_val": "sunken_heli" }, "target_params": { "om_terrain": "sunken_heli", "z": -5 } },
```
Was this actually finding it?